packages/strapi-plugin-users-permissions/admin/src/containers/Roles/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Since you will need a router a suggest the following modifications:

```js
import React from 'react';
import { Switch, Route, Link } from 'react-router-dom';
import { useGlobalContext, CheckPagePermissions, NotFound } from 'strapi-helper-plugin';
import pluginId from '../../pluginId';
import pluginPermissions from '../../permissions';

const RolesEditPage = () => <div>Role edit</div>;

const ProtectedRolesEditPage = () => (
  <CheckPagePermissions permissions={pluginPermissions.updateRole}>
    <RolesEditPage />
  </CheckPagePermissions>
);
const RolesCreatePage = () => (
  <CheckPagePermissions permissions={pluginPermissions.createRole}>
    <RolesEditPage />
  </CheckPagePermissions>
);

const RolesListPage = () => {
  return (
    <CheckPagePermissions permissions={pluginPermissions.accessRoles}>
      <div>Role list</div>
      <div>
        <Link to="/settings/users-permissions/roles/1">Edit Role</Link>
      </div>
      <div>
        <Link to="/settings/users-permissions/roles/new">Create Role</Link>
      </div>
    </CheckPagePermissions>
  );
};

const Role = () => {
  const { settingsBaseURL } = useGlobalContext();

  return (
    <Switch>
      <Route path={`${settingsBaseURL}/${pluginId}/roles/new`} component={RolesCreatePage} exact />
      <Route
        path={`${settingsBaseURL}/${pluginId}/roles/:id`}
        component={ProtectedRolesEditPage}
        exact
      />
      <Route path={`${settingsBaseURL}/${pluginId}/roles`} component={RolesListPage} exact />
      <Route path="" component={NotFound} />
    </Switch>
  );
};

export default Roles;
```